packages/TorneloScoresheet/src/components/ChessBoard/ChessBoard.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { View } from 'react-native';
+import { BoardPosition } from '../../types/ChessMove';
+import ChessSquare from '../ChessSquare/ChessSquare';
+import RoundedView from '../RoundedView/RoundedView';
+import { styles } from './style';
+
+export type ChessBoardProps = {
+  boardPositions: BoardPosition[][];
+  flipBoard: boolean;
+};
+
+const ChessBoard: React.FC<ChessBoardProps> = ({
+  boardPositions,
+  flipBoard,
+}) => {
+  if (flipBoard) {
+    boardPositions = boardPositions.reverse();
```

Comment:
This reverses the array in-place. It is bad practice to mutate a React prop (as this can have unexpected behaviour for the parent).

Two options here - reverse the rendering with `flex-direction: 'row-reverse'`, or, take a copy:

```ts
{(flipBoard ? [...boardPositions].reverse() : boardPositions).map(...)};
```
(Or something similar)